README.md

```diff
@@ -1,2 +1,7 @@
 # ci-oracle-client-runner
-A container that contains an oracle client for running against Concourse 
+A container that contains an oracle client for running against Concourse.\
+The files here are based on the OracleInstantClient docker images provided by Oracle, that can be found here: https://github.com/oracle/docker-images/tree/master/OracleInstantClient \
```

Comment:
The information available at this URL is likely to change as it contains a mutable reference (i.e. `master`). Links to GitHub should ideally be permalinks or they quickly become outdated. Press `y` when viewing the page on GitHub to update the URL to a permalink and replace the above URL.

Reply:
Done.